pipeline/metadata/5.2.yaml

```diff
@@ -0,0 +1,152 @@
+#=====================================================================================
+# Metadata file for 5.2 RHCS release version.
+# Single file to execute the all tier-1, tier-2, tier-3 and tier-4 suites in parallel
```

Comment:
This can be changed as "Single file to specify test suites to be executed for all regression (perbuild) and periodical tests"

Reply:
@manasagowri I made necessary changes, please review